src/components/containers/TransformAccordion.js

```diff
@@ -83,9 +83,17 @@ class TransformAccordion extends Component {
         </TransformFold>
       ));
 
+    // cannot have 2 Split transforms on one trace:
+    // https://github.com/plotly/plotly.js/issues/1742
+    const addActionOptions =
+      container.transforms &&
+      container.transforms.some(t => t.type === 'groupby')
+        ? transformTypes.filter(t => t.type !== 'groupby')
+        : transformTypes;
+
```

Comment:
I don't think you need to 'some' it, just go straight to 'filter' - some takes performance too!